arch/arm/soc/nordic_nrf/nrf51/Kconfig.series

```diff
@@ -17,5 +17,6 @@ config  SOC_SERIES_NRF51X
 	select XIP
 	select HAS_CMSIS
 	select HAS_NRFX
+	select HAS_SEGGER_RTT
```

Comment:
This was missing from nRF51; just to double-check that this is the indented behavior, now.